public/assets/js/spongedownloads.js

```diff
@@ -0,0 +1,388 @@
+const API = "https://dev.minecrell.net/downloads"
```

Comment:
That is not that easy because this is a JavaScript file, we can't easily modify it from the Go app.